### PR TITLE
LVNOD-27 Added model user without controllers and routes

### PR DESCRIPTION
**Pull request by yarvanya**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-27)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-43)

## Code reviewers

- [x] @yurii-kokhalevych
- [x] @denisbodnar
- [x] @AlbaMontis

### Second Level Review

- [ ] @lhalam
- [ ] @4asof4ik

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo
## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: config/env.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  DATABASE_URL: process.env.DATABASE_URL || 'postgresql://localhost:5432/dami_database',
+  DATABASE_NAME: process.env.DATABASE_NAME || 'dami_database',
+  DATABASE_HOST: process.env.DATABASE_HOST || 'localhost',
+  DATABASE_USERNAME: process.env.DATABASE_USERNAME || 'postgres',
+  DATABASE_PASSWORD: process.env.DATABASE_PASSWORD || 'admin',
+  DATABASE_PORT: process.env.DATABASE_PORT || 5432,
+  DATABASE_DIALECT: process.env.DATABASE_DIALECT || 'postgres',
+  NODE_ENV: process.env.NODE_ENV || 'development'
+};
+module.exports = env;
```
Comment (denisbodnar) [inline config/env.js]: add a new line

File: server/migrations/20170520152757-create_table_user.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+         }, 
+         status_user_state: { 
+           type: Sequelize.STRING
+         }
+    });
+  },
+  down: function (queryInterface, Sequelize) {
+    return queryInterface.dropTable('users');
+  }
+};
```
Comment (denisbodnar) [inline server/migrations/20170520152757-create_table_user.js]: add a new line

File: server/seeders/user.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        profile_id: 2,
+        is_activate: true,
+        status_user_state: null
+      }
+    ], {});
+  },
+  down: function (queryInterface, Sequelize) {
+    return queryInterface.bulkDelete('users', null, {});
+  }
+};
```
Comment (denisbodnar) [inline server/seeders/user.js]: add a new line

File: server/models/user.model.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        status_user_state: { 
+            type: DataTypes.STRING 
+        } 
+        }, { 
+            timestamps: false,
+            paranoid: true,
+            underscored: true 
+        });
+    return User;
+};
```
Comment (denisbodnar) [inline server/models/user.model.js]: add a new line

**Review by denisbodnar — DISMISSED** [review]

File: config/env.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+const env = {
+  PORT: process.env.PORT || 8090,
+  DATABASE_URL: process.env.DATABASE_URL || 'postgresql://localhost:5432/dami_database',
+  DATABASE_NAME: process.env.DATABASE_NAME || 'dami_database',
+  DATABASE_HOST: process.env.DATABASE_HOST || 'localhost',
+  DATABASE_USERNAME: process.env.DATABASE_USERNAME || 'postgres',
+  DATABASE_PASSWORD: process.env.DATABASE_PASSWORD || 'admin',
```
Comment (AlbaMontis) [inline config/env.js]: Is it secure?

File: server/models/user.model.js (hunk 4 above)
Comment (yurii-kokhalevych) [reply server/models/user.model.js]: Different validate "len" in password in model and migration

File: server/models/user.model.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+    const User = sequelize.define('user', {
```
Comment (AlbaMontis) [inline server/models/user.model.js]: Too many spaces.

**Review by yurii-kokhalevych — DISMISSED** [review]

File: server/models/user.model.js [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+          type: DataTypes.INTEGER
+        }, 
+        is_activate: { 
+          type: DataTypes.BOOLEAN, 
+          allowNull: false 
+        }, 
+        status_user_state: { 
+            type: DataTypes.STRING 
+        } 
+        }, {
```
Comment (AlbaMontis) [inline server/models/user.model.js]: Too many spaces. Fix indentation in this file.

File: config/env.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+  DATABASE_PORT: process.env.DATABASE_PORT || 5432,
+  DATABASE_DIALECT: process.env.DATABASE_DIALECT || 'postgres',
+  NODE_ENV: process.env.NODE_ENV || 'development'
+};
+module.exports = env;
```
Comment (AlbaMontis) [inline config/env.js]: Not space but new empty line.

**Review by shyyka — APPROVED** [review]

**Review by shyyka — APPROVED** [review]

**Review by denisbodnar — APPROVED** [review]

**Review by AlbaMontis — APPROVED** [review]

**Review by yurii-kokhalevych — APPROVED** [review]

**Pull request closed without merging**